use-timescale/continuous-aggregates/refresh-policies.md

```diff
@@ -59,10 +59,10 @@ See the [API reference][api-reference] for the full list of required and optiona
 </Procedure>
 
 The policy in this example ensures that all the data in the continuous aggregate
-is up to date with the hypertable except for anything written in the last hour.
-It also does not refresh the last time bucket of the continuous aggregate.
+is up to date with the hypertable, except for anything written within the last hour. This means that the actual `INSERT` or `UPDATE` of the data falls within the last hour, regardless of its timestamp. The policy also does not refresh the last time bucket of the continuous aggregate.
```

Comment:
Not sure I understand the thing about all the inserted and updated data falling within the last hour regardless of timestamp.

Looking at the policy, it seems to:

- Run once every hour
- when it runs, it materializes all buckets with outdated data except those that fall within the last hour

Reply:
Ah sorry, here is the discussion we had about this:  https://iobeam.slack.com/archives/CEKV5LMK3/p1748335245838269 Does this add context?